### PR TITLE
Detailed progressbar

### PR DESCRIPTION
**Pull request by jubianchi**
capture d ecran 2015-01-12 a 00 04 05

### CODE REVIEW

**Comment by youknowriad** [comment]
+1

**Comment by willgarcia** [comment]
+1

**Comment by jubianchi** [comment]
\o/

File: server/app/index.js [hunk 1]
```diff
@@ -74,19 +74,37 @@ bedoon.app.get('/dashboard', function(req, res) {
...
+                progressCurrent: 0,
+                progressBlocked: 0
             };
 
             bedoon.models.project_delivery.find({delivery: delivery.id}, function(err, project_deliveries) {
                 project_deliveries.forEach(function(project_delivery) {
                     if (project_delivery.status !== 'delivered') {
                         delivery.ready = false;
+
+                        if (project_delivery.status === 'planned') {
```
Comment (esion) [inline server/app/index.js]: Switch case ?

**Comment by esion** [comment]
+1

**Pull request merged**